v63004/inref/gtm8699.m

```diff
@@ -0,0 +1,26 @@
+;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;
+;								;
+; Copyright (c) 2018 YottaDB LLC. and/or its subsidiaries.	;
+; All rights reserved.						;
+;								;
+;	This source code contains the intellectual property	;
+;	of its copyright holder(s), and is made available	;
+;	under a license.  If you do not know the terms of	;
+;	the license, please stop and do not read further.	;
+;								;
+;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;
+;
+gtm8699()
+
+	WRITE "    $VIEW(""STATSHARE"",""DEFAULT"")",!
+	WRITE "    "
+	WRITE $VIEW("STATSHARE","DEFAULT"),!
+
+	WRITE "    VIEW ""STATSHARE"" ",!
+	VIEW "STATSHARE"
+
+	WRITE "    $VIEW(""STATSHARE"",""DEFAULT"")",!
+	WRITE "    "
+	WRITE $VIEW("STATSHARE","DEFAULT"),!
+
+	quit
```

Comment:
The following note is from the release note. The bolded part needs to be tested too.

For the process to be sharing, **the database must be enabled for sharing** and the process must have opted in to share.